src/main/java/com/box/sdk/BoxItem.java

```diff
@@ -36,6 +36,11 @@ public BoxItem(BoxAPIConnection api, String id) {
     }
 
     /**
+     * @return base URL for current item
```

Comment:
This name is confusing for two reasons.  One is the same name as a method on the API class, and two it isn't really returinging a base URL. It is returning a base URL plus an item specifier.   I suggest we change the name getItemURL and change the javadocs to reflect what the method really returns.